doc/backend/branding.md

```diff
@@ -153,28 +151,54 @@ cp frontend/dist/assets/branding.sample/app_custom_actions.json frontend/dist/as
 
 Edit the file `frontend/dist/assets/branding/app_custom_actions.json`.
 
-It is a json file containing a list of objects. Each objects are the definition of an App Custom Actions.
-
 Complete example of `frontend/dist/asset/branding/app_custom_actions.json`:
 ```json
-[{
-  "icon_text": "fas fa-chess-queen",
-  "icon_image": "",
-  "content_type_filter": "file",
-  "content_extension_filter": ".jpg,.png",
-  "content_label_regex_filter": "field report",
-  "workspace_id_filter": "1,2,3",
-  "user_role_filter": "workspace-manager,content-manager",
-  "user_profile_filter":"administrators,trusted-users",
-  "label": {
-    "fr": "Ouvrir la maquette dans SomeSoftware",
-    "en": "Open mockup in SomeSoftware"
-  },
-  "link": "https://some.software.com/open?content={content.content_id}&label={content.label}&space_id={content.workspace_id}&user_id={user.user_id}"
-}]
+{
+  "user_sidebar_dropdown": [],
+  "user_sidebar_shortcuts": [],
+  "content_in_list_dropdown": [],
+  "content_app_dropdown": [{
+    "icon_text": "fas fa-chess-queen",
+    "icon_image": "",
+    "content_type_filter": "file",
+    "content_extension_filter": ".jpg,.png",
+    "content_label_regex_filter": "field report",
+    "workspace_id_filter": "1,2,3",
+    "user_role_filter": "workspace-manager,content-manager",
+    "user_profile_filter":"administrators,trusted-users",
+    "label": {
+      "fr": "Ouvrir la maquette dans SomeSoftware",
+      "en": "Open mockup in SomeSoftware"
+    },
+    "link": "https://some.software.com/open?content={content.content_id}&label={content.label}&space_id={content.workspace_id}&user_id={user.user_id}"
+  }],
+  "space_dashboard_action_list": []
+}
```

Comment:
There are a lot of filters in this example and it might be best to show an example easier to implement, maybe one without much filter to be tested easily by the user